mw-webapp/src/service/WayService.ts

```diff
@@ -47,12 +48,25 @@ export class WayService {
    * Get WaysDTO of user mentoring ways
    */
   public static async getMentoringWaysDTO(uuid: string): Promise<WayDTO[]> {
-    const mentoringWaysRef = collection(db, PATH_TO_WAYS_COLLECTION);
-    const mentoringWaysQuery = query(mentoringWaysRef, where("currentMentorUuids", "array-contains", uuid));
+    const waysRef = collection(db, PATH_TO_WAYS_COLLECTION);
+    const mentoringWaysQuery = query(waysRef, where("currentMentorUuids", "array-contains", uuid));
     const mentoringWaysRaw = await getDocs(mentoringWaysQuery);
     const mentoringWays = querySnapshotToDTOConverter<WayDTO>(mentoringWaysRaw);
 
     return mentoringWays;
   }
 
+  /**
+   * Get WaysDTO of user favourite ways
+   */
+  public static async getFavouriteWaysDTO(uuid: string): Promise<WayDTO[]> {
+    const userDTO = await UserService.getUserDTO(uuid);
+    const waysRef = collection(db, PATH_TO_WAYS_COLLECTION);
+    const favouriteWaysQuery = query(waysRef, where("uuid", "in", userDTO.favoriteWayUuids));
```

Comment:
On line 65, there are two spellings of the word "favorite". let's choose one variant

Reply:
Changed to favorite everywhere